src/Plugin/EntityReferenceSelection/OgRoleSelection.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@jludwig 

----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 22 | ERROR | [x] Data types in @return tags need to be fully
    |       |     namespaced
----------------------------------------------------------------------